Tools-Override/codeAnalysis.targets

```diff
@@ -11,16 +11,12 @@
 
   <!-- PInvokeChecker data files-->
   <PropertyGroup Condition="'$(OSGroup)'=='Windows_NT' and '$(EnableDotnetAnalyzers)' == 'true'">
-    <!-- 
-       This file helps us choose the correct allowed pinvoke file.
-       1. In case we are building for windows and TargetGroup is Netcore50 or netcore50aot (Store profile) or SupportsUWP is set to true we use OneCoreUWPApis.txt
-       2. For all other windows configuration we use OneCoreAPIs.txt file.
-    -->
-    <UseWin32Apis Condition="'$(OSGroup)'=='Windows_NT'">true</UseWin32Apis>
-    <ValidPinvokeMappings>$(MSBuildThisFileDirectory)PinvokeAnalyzer_OneCoreApis.txt</ValidPinvokeMappings>
-    <ValidPinvokeMappings Condition="'$(UseWin32Apis)'=='true'">$(MSBuildThisFileDirectory)PinvokeAnalyzer_Win32Apis.txt</ValidPinvokeMappings>
-    <ValidPinvokeMappings Condition="'$(TargetGroup)'=='netcore50' or '$(TargetGroup)'=='netcore50aot' or '$(SupportsUWP)'=='true'">$(MSBuildThisFileDirectory)PinvokeAnalyzer_UWPApis.txt</ValidPinvokeMappings>
-    <ValidPinvokeMappings Condition="('$(TargetGroup)'=='netcore50' or '$(TargetGroup)'=='netcore50aot' or '$(SupportsUWP)'=='true') and '$(UseWin32Apis)'=='true'">$(MSBuildThisFileDirectory)PinvokeAnalyzer_Win32UWPApis.txt</ValidPinvokeMappings>
+    <!-- Just validate for OneCore: default -->
+    <ValidPinvokeMappings>$(MSBuildThisFileDirectory)PinvokeAnalyzer_Win32Apis.txt</ValidPinvokeMappings>
```

Comment:
Nit: Correct casing would be PInvoke, but I guess you cannot fix it in this change because of part of it comes from buildtools?